dll/win32/shdocvw/shdocvw_main.c

```diff
@@ -31,6 +31,7 @@
 #ifdef __REACTOS__
 #include "winnls.h"
 #include <shlguid_undoc.h>
+#include <rpcproxy.h>
```

Comment:
For which function(s) is this include needed?

Reply:
For `__wine_register_resources` and `__wine_unregister_resources`.